mne/filter.py

```diff
@@ -1402,8 +1401,7 @@ def resample(x, up=1., down=1., npad=100, axis=-1, window='boxcar', n_jobs=1,
         Axis along which to resample (default is the last axis).
     %(window-resample)s
     %(n_jobs-cuda)s
-    %(pad-fir)s
-        The default is ``'reflect_limited'``.
+    %(pad)s
```

Comment:
Yes, isn't the guideline to not provide default in docstring whenever possible since it's already explicitly given above the docstring in the function definition? This default is not present everywhere anyway..

I just kept it for `raw.resample()` and `FilterMixin.resample()` that have different default padding values (`edge` vs `reflect_limited`).

Reply:
You mean even if it's in the function signature? We've been transitioning to putting / trying to put it in the docstring as well. For long docstrings it's easier for people than having to look back up at the signature line to see the default.

Reply:
Alright, reverting this then.